qubes-rpc-policy/80-whonix.policy

```diff
@@ -1,13 +1,13 @@
-# service            arg source            target            action params
+# service            arg       source            target            action params
 
-whonix.SdwdateStatus  *  @tag:anon-gateway @tag:anon-vm      allow autostart=no
-whonix.SdwdateStatus  *  sys-whonix        @tag:anon-vm      allow autostart=no
-whonix.SdwdateStatus  *  @anyvm            @anyvm            deny
+whonix.SdwdateStatus +         @tag:anon-gateway @tag:anon-vm      allow  autostart=no notify=no
+whonix.SdwdateStatus +         @tag:anon-gateway @default          deny   notify=no
+whonix.SdwdateStatus +         @anyvm            @anyvm            deny
 
-whonix.NewStatus      *  @tag:anon-vm      @tag:anon-gateway allow autostart=no
-whonix.NewStatus      *  @tag:anon-vm      sys-whonix        allow autostart=no
-whonix.NewStatus      *  @anyvm            @anyvm            deny
+whonix.NewStatus     *         @tag:anon-vm      @tag:anon-gateway allow  autostart=no
+whonix.NewStatus     *         @anyvm            @anyvm            deny
 
-whonix.GatewayCommand *  @tag:anon-gateway @tag:anon-vm      allow autostart=no
-whonix.GatewayCommand *  sys-whonix        @tag:anon-vm      allow autostart=no
-whonix.GatewayCommand *  @anyvm            @anyvm            deny
+whonix.GatewayCommand +restart @tag:anon-gateway @tag:anon-vm      allow  autostart=no
+whonix.GatewayCommand +stop    @tag:anon-gateway @tag:anon-vm      allow  autostart=no
+whonix.GatewayCommand +showlog @tag:anon-gateway @tag:anon-vm      allow  autostart=no
```

Comment:
```suggestion
whonix.GatewayCommand +restart @tag:anon-gateway @tag:anon-vm      allow  autostart=no notify=no
whonix.GatewayCommand +stop    @tag:anon-gateway @tag:anon-vm      allow  autostart=no notify=no
whonix.GatewayCommand +showlog @tag:anon-gateway @tag:anon-vm      allow  autostart=no notify=no
```

These suffer from the same race condition as `whonix.SdwdateStatus` unless I have missed something.

Reply:
I couldn't reproduce this yet. These commands are only sent manually if the user clicks A) restart sdwdate, B) stop sdwdate or C) show sdwdate log for an App Qube such as anon-whonix.
(sys-whonix doesn't need qrexec to do that for sys-whonix.)

So this could only be a race condition if the user manages to `qvm-kill anon-whonix`, this going unnoticed by sdwdate-gui on `sys-whonix` and the user then attempting to use A), B) or C).

"this going unnoticed by sdwdate-gui" - there is currently no such bug as far as I am aware.

My idea was "in doubt, don't use `notify=no`" for now.

Reply:
Not a big deal either way. Should anything be broken, I guess one would have to fall back to dom0 command line anyhow:

    sudo journalctl -f | grep qrexec